source/usage-examples/replaceOne.txt

```diff
@@ -80,7 +80,7 @@ The following snippet uses the following objects and methods:
 .. literalinclude:: /includes/usage-examples/code-snippets/ReplaceOne.java
    :language: java
 
-If you are using the **legacy API**, see (TODO: link to FAQ) to determine what
+If you are using the **legacy API**, see the :doc:`FAQ </faq>` to determine what
```

Comment:
This info doesn't seem to be in the FAQ yet. Consider hiding this paragraph until it is.

Reply:
Thanks for checking so thoroughly. This is definitely planned for the FAQ which is where the Java Drivers Team wanted it. Added a sentence in the ticket to make sure https://jira.mongodb.org/browse/DOCSP-9604 it happens.